SPECS/libxslt/libxslt.spec

```diff
@@ -25,6 +26,7 @@ Header files for doing development with libxslt.
 
 %prep
 %setup -q
+%patch0 -p1
```

Comment:
Consider using `%autosetup -p1` instead of %setup and %patch, so future patches can be easily added without modifying the extraction process